.changeset/serious-games-add.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Since the Atlas site is getting updated, `@microsoft/atlas-site` needs a changeset as well.